extensions/applicationinsights-dependencies-js/src/ajax.ts

```diff
@@ -307,6 +307,11 @@ export interface IDependenciesPlugin extends IDependencyListenerContainer {
      * @param dependencyData - dependency data object
      */
     trackDependencyData(dependency: IDependencyTelemetry): void;
+
+    /**
+     * Resets the ajax attempts counter. This is typically called on page view to allow a fresh set of ajax calls to be tracked.
+     */
+    resetAjaxAttempts?(): void;
```

Comment:
The tests are failing because the getPlugin() call expects the "type" to extend IPlugin, evaluate changing this interface to also extend IPlugin and in AnalyticsPlugin change the getPlugin to use getPlugin<IDependenciesPlugin>